src/Symfony/Component/Routing/Tests/RouteCompilerTest.php

```diff
@@ -36,15 +36,15 @@ public function provideCompileData()
             array(
                 'Static route',
                 array('/foo'),
-                '/foo', '#^/foo$#s', array(), array(
+                '/foo', '#^/foo$#us', array(), array(
```

Comment:
a lot of tests redesigned to check for utf-8 – is there any test checking other encodings?